roles/ces_common/tasks/configure.yml

```diff
@@ -21,9 +21,9 @@
 
 - name: configure | Prepare server nodes string
   set_fact:
-   scale_server_nodes: "{{ scale_server_nodes + ',' + item|string }}"
+    scale_server_nodes: "{{ scale_server_nodes + ',' + hostvars[item]['scale_daemon_nodename'] | string }}"
```

Comment:
@acch Can we keep `inventory_hostname `  instead of  `scale_daemon_nodename` ?

Reply:
In the next task we'll run `mmchlicense ... -N`, which will require the nodename as opposed to the inventory name.